### ISSUE TITLE
WCAG review of GEO.ca

### ISSUE THREAD

**Issue by sean-eagles** [Canadian-Geospatial-Platform/geo.ca#109]
Getting a complete review of GEO.ca to ensure we are following accessibility standards.  This will be performed by Vlad.

**Comment by elatsis** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Canadian-Geospatial-Platform-62d04ca87c442300145c51fa/issues/Canadian-Geospatial-Platform/geo.ca/109?planning-poker) with Zenhub @sean-eagles

**Comment by elatsis** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Canadian-Geospatial-Platform-62d04ca87c442300145c51fa/issues/Canadian-Geospatial-Platform/geo.ca/109?planning-poker) with Zenhub @vgulyy

**Comment by sean-eagles** [comment]
Done as part of the UX review.  Vlad please send this to Jared at your earliest convenience.

**Comment by vgulyy** [comment]
The doc is now with Jared

**Comment by sean-eagles** [comment]
WCAG assessment going further, and implementation to be done by Jared where possible.

**Comment by jaredkinger** [comment]
I reviewed geo.ca's accessibility using the Lighthouse test, Wave test and through manual testing. Below is a summary of the issues that were found and the number instances for each issue:

- 5 x Images missing alt text
- 4 x Heading elements are not in a sequentially-descending order
- 8 x Form elements missing a label
- 2 x iframe missing title
- 14 x keyboard navigation errors
- 22 x keyboard focus errors
